drivers/cdc_demo_client/cdc_demo.c

```diff
@@ -3,83 +3,234 @@
  *
  * cdc demo - Example of using: USB Communication Device Class
  *
- * Copyright 2019 Phoenix Systems
- * Author: Hubert Buczynski
+ * Copyright 2019, 2021 Phoenix Systems
+ * Author: Hubert Buczynski, Gerard Swiderski
  *
  * This file is part of Phoenix-RTOS.
  *
  * %LICENSE%
  */
 
 
-#include <errno.h>
 #include <stdio.h>
+#include <stdlib.h>
 #include <string.h>
+#include <time.h>
 #include <unistd.h>
+#include <cdc_client.h>
 
-#include <sys/platform.h>
-#include <phoenix/arch/imxrt.h>
 
-#include <cdc_client.h>
+#define TIME_ONESECOND          1000000000ULL
+#define TIME_REALTIME_OVERFLOW  (TIME_ONESECOND / 5)
+
+#define USB_EP_BULK_EP          2
```

Comment:
`USB_EP_BULK_EP` depends on endpoint definition in **cdc-client.c** . Currently the code is cleaning up so it's a good moment to move this definition to **cdc_client.h** and link it with endpoint definition.

Reply:
Done.